com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/steps/CreateOrUpdateAppStep.java

```diff
@@ -246,127 +276,104 @@ public void printStepEndMessage() {
             getStepLogger().debug(Messages.APP_UPDATED, app.getName());
         }
 
-    }
-
-    private UpdateState updateApplicationEnvironment(CloudApplicationExtended app, CloudApplication existingApp,
-                                                     CloudControllerClient client,
-                                                     CloudApplicationExtended.AttributeUpdateStrategy applicationAttributesUpdateStrategy) {
-        return new EnvironmentApplicationAttributeUpdater(existingApp,
-                                                          getUpdateStrategy(applicationAttributesUpdateStrategy.shouldKeepExistingEnv()),
-                                                          getStepLogger()).updateApplication(client, app);
-    }
-
-    private void reportApplicationUpdateStatus(CloudApplicationExtended app, boolean appPropertiesChanged) {
-        if (!appPropertiesChanged) {
-            getStepLogger().info(Messages.APPLICATION_UNCHANGED, app.getName());
-            return;
+        private UpdateState updateApplicationEnvironment(CloudApplicationExtended app, CloudApplication existingApp,

```

Comment:
Update your formatter. You can get the latest one from [here](https://github.com/cloudfoundry-incubator/multiapps/tree/master/ide).